components/atoms/LoadingDots.jsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import trimString from '../utils/trimString';
+import cx from 'classnames';
 
-const LoadingDots = ({ whiteDots = false, size, className, style }) => {
-    const baseClass = 'gds-loading__dot',
-        whiteDotsClass = whiteDots ? `${baseClass}--white` : '',
-        sizeClass = size ? `${baseClass}--${size}` : '';
-
-    const classNames = trimString(`${baseClass} ${whiteDotsClass} ${sizeClass}`);
+const LoadingDots = ({ whiteDots, size, className, style }) => {
```

Comment:
oh yeah i noticed on this component, there's a padding that is applied on the story that should be removed

Reply:
oh good catch. Actually I was applying `className` twice in LoadingDots.